bin/real

```diff
@@ -1 +0,0 @@
-real.armv7l
```

Comment:
why delete this?

Reply:
Revert this changes for now. When got time, will test it on firewalla red.